src/azure-cli/azure/cli/command_modules/monitor/operations/action_groups.py

```diff
@@ -225,7 +226,8 @@ def _build_arguments_schema(cls, *args, **kwargs):
         args_schema.receiver_actions = AAZCustomListArg(
             options=["--add-actions"],
             singular_options=["--add-action", "-a"],
-            help="Add receivers to the action group." + '''
+            help='''
+        Add receivers to the action group.
         Usage:   --add-action TYPE NAME [ARG ...]
```

Comment:
Please also fix line 361 in this file for doc formatting:
Thanks
```
help='''
        Add receivers to the action group.
        Usage:   --add-action TYPE NAME [ARG ...]
```

Reply:
committed fix